### PR TITLE
HandleDelete functions added

### PR DESCRIPTION
**Pull request by bibang007**
Functions to delete Thread, Category and Comment in app.js

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -45,6 +59,18 @@ class App extends Component {
     })
   }
 
+  //delete Category function
+
+  async handleDeleteCategories(event) {
+    event.preventDefault();
+    await fetch(`${url}/category/:id`, {
```
Comment (imanipaul) [inline client/src/App.js]: could you add ${event.target.id} instead of just 'id'?

File: client/src/App.js [hunk 2]
```diff
@@ -56,6 +82,18 @@ class App extends Component {
     })
   }
 
+  // Delete Comment function
+
+  async handleDeleteComments(event) {
+    event.preventDefault();
+    await fetch(`${url}/comment/:id}`, {
```
Comment (imanipaul) [inline client/src/App.js]: same here.

**Review by imanipaul — CHANGES_REQUESTED** [review]
Looks good, just need to edit one thing

File: client/src/App.js (hunk 2 above)
Comment (bibang007) [reply client/src/App.js]: changes made

**Review by imanipaul — DISMISSED** [review]
looks good

**Review by imanipaul — APPROVED** [review]
great

**Pull request merged**